react/components/FilterSidebar.js

```diff
@@ -243,6 +244,7 @@ const FilterSidebar = ({
             showClearByFilter={showClearByFilter}
             updateOnFilterSelectionOnMobile={updateOnFilterSelectionOnMobile}
             priceRangeLayout={priceRangeLayout}
+            onClose={handleClose}
```

Comment:
Was intentional? This may change the current behavior, right?

Reply:
Sorry about this. I left it by mistake. Initially there were more changes.